docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,333 @@
+Datasets
+==========
+
+Revisiting the Boston Housing Dataset
+-------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The Boston Housing dataset is one of three datasets currently callable in 
+fairlearn's *datasets* module. In the past, it has commonly been used for 
+benchmarking in popular machine learning libraries, including scikit-learn and 
+open-ml. However, as the machine learning community has developed awareness 
+about fairness issues with popular benchmarking datasets, the Boston Housing 
+data has been phased out of many libraries. We migrated the dataset to  
+fairlearn after it was phased out of scikit-learn in June 2020. The dataset  
+remains in fairlearn as an example of how system racism can occur in data  
+and to show the effect of fairlearn's assessment and pre-/in-/post-processing  
+tools on real, problematic data. 
+
+**We wrote this blog post to achieve the following goals:**
+  * Educate users about the history of the dataset and its bias
+  * Show users how bias manifests in the data and in downstream modelling tasks
+  * Suggest best practices for dealing with the Boston Housing data and 
+  alternative benchmarking datasets
+
+.. _dataset_origin:
+
+Dataset Origin and Use
+^^^^^^^^^^^^^^^^^^^^^
+
+Contrary to expectation, the Boston Housing dataset was not developed for 
+economic purposes. `Harrison and Rubenfield (1978) <https://deepblue.lib.umich.edu/bitstream/handle/2027.42/22636/0000186.pdf?sequence=1&isAllowed=y>` 
```

Comment:
I think links require the following syntax
```
`text <link>`_
```
The underscore seems to be missing. You can check if it works by building the docs locally. Perhaps I'm wrong, but you may want to check all the links in your PR.

Reply:
Oh ok, I will add the underscore throughout!